metadata-ingestion/examples/mce_files/bootstrap_mce2.json

```diff
@@ -0,0 +1,1974 @@
+[
+  {
+    "auditHeader": null,
```

Comment:
What's the difference between this and the original bootstrap file?

Reply:
This was for testing-- will remove.